content/docs/coldchain/equipment.md

```diff
@@ -82,34 +82,68 @@ template which can be used to import assets.
+| **Catalogue item code** | The [Asset Catalogue code](/docs/catalogue/assets) - i.e. the kind of asset to be imported | E003/002                   | \*                |
+| **Installation date**   | The date that the asset was installed at the Store                                         | 2024-01-14                 |                   |
+| **Replacement due**     | The date that the asset is due for replacement                                             | 2026-01-14                 |                   |
+| **Warranty start**      | The start date of the asset's warranty                                                     | 2024-01-14                 |                   |
+| **Warranty end**        | The end date of the asset's warranty                                                       | 2025-01-14                 |                   |
+| **Serial Number**       | The serial number of that particular asset                                                 | 123456789ABC               |                   |
+| **Status**              | The asset's [functional status](#status-history)                                           | Functional                 |                   |
+| **Needs replacement**   | Whether the asset needs replacing                                                          | TRUE                       |                   |
+| **Notes**               | General notes about the asset                                                              | Kept in EPI office         |                   |
+| **Asset properties**    | Many columns may be also be available here for configured asset properties                 | energy_source: Electricity |                   |
+
+When you are ready, drag and drop the csv file into the modal or click on the `Browse files` button to select the file from your computer.
 
 <p align="center">
     <img src="/docs/coldchain/images/equipment_import_upload.png" width="750" height="420">
 </p>
 
+#### If things go wrong
+
+Sometimes errors occur when importing CCE assets. Here are some common errors you might encounter:
+
+##### Upload Errors
+
+After uploading your CSV file, your data will be validated and displayed to you to review. At this step, if the data in your uploaded file is not valid, you will see an error message and won't be able to continue to the import step.
+
+![](/docs/coldchain/images/cce_import_error.png)
+
+Common mistakes here are uploading a file type other than CSV, or if you included some duplicate asset numbers in your file. You'll see these errors in the `Error message` column.
+
+You can use the `Export` button at the bottom of the upload window to download a CSV file which includes the error messages, to investigate what went wrong.
+
+Once you have corrected the errors, you can go back to the `Upload` tab to upload the corrected file.
```

Comment:
I think it's almost worth having a note about date formats, as that's the one that generally gets automatically mangled by spreadsheets?

Reply:
Yeah good idea! Will add :pray: